client/src/components/Home.tsx

```diff
@@ -92,7 +150,38 @@ function Home() {
           </WebPlaybackContainer>
         }
       </MainContainer>
-    </div>
+      {
+        dialogOpen === 'create' &&
+        <TextInput
+          isDialogOpen={dialogOpen === 'create'}
+          handleDialogClose={handleDialogClose}
+          inputValue={newPlaylistTitle}
+          inputLabel="Playlist Title"
+          handleInputChange={(event) => setNewPlaylistTitle(event.target.value)}
+          handleSubmit={handleCreatePlaylist}
+          formTitle="Create New Playlist"
+        />
+      }
+      {
+        dialogOpen === 'edit' &&
```

Comment:
I just copied and pasted from the create version -- will update this in a following PR when we actually add editing abilities